rerun_cpp/src/rerun/component_batch_adapter.hpp

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+namespace rerun {
+    /// The ComponentBatchAdaptor trait is responsible for mapping an input argument to a
+    /// ComponentBatch.
+    ///
+    /// There are default implementations for standard containers of components, as well as single
+    /// components.
```

Comment:
```suggestion
    /// components. These can be found in `rerun/component_batch_adapter_builtins.hpp`.
```

Reply:
ah. Yes! Being able to link to the file was one of the reasons I wanted it separate!